checkstyle-style.xml

```diff
@@ -13,10 +13,8 @@
             <property name="severity" value="error" />
         </module>
 
-	    <!-- Make the @SuppressWarnings annotations available to Checkstyle -->
-	    <module name="SuppressWarningsHolder" />
-
-        <module name="FileContentsHolder" />
```

Comment:
`FileContentsHolder` is removed from [new release](http://checkstyle.sourceforge.net/releasenotes.html#Release_8.2). It is not required any more and just need to be removed from clients configurations.